sdk/containerservice/azure-mgmt-containerservice/tests/test_mgmt_containerservice.py

```diff
@@ -19,29 +19,68 @@ def setUp(self):
         self.cs_client = self.create_mgmt_client(
             azure.mgmt.containerservice.ContainerServiceClient
         )
+        if self.is_live:
+            from azure.mgmt.network import NetworkManagementClient
+            self.network_client = self.create_mgmt_client(
+                NetworkManagementClient
+            )
+    
+    def create_subet(self, group_name, location):
+        # Create virtual network
+        self.network_client.virtual_networks.create_or_update(
+            group_name,
+            "vnet",
+            {
+                "address_space": {
+                    "address_prefixes": [
+                    "10.0.0.0/8"
+                    ]
+                },
+                "location": location 
+            }
+        ).result()
+
+        # Create subnet
+        subnet = self.network_client.subnets.create_or_update(
+            group_name,
+            "vnet",
+            "v-subnet",
+            {
+                "address_prefix": "10.0.0.0/8"
+            }
+        ).result()
+
+        return subnet.id
 
-    @ResourceGroupPreparer(location='westus2')
+    @ResourceGroupPreparer(location='westus')
     def test_container(self, resource_group, location):
         container_name = self.get_resource_name('pycontainer')
+
+        if self.is_live:
+            subnet_id = self.create_subet(resource_group.name, location)
+        else:
+            subnet_id = "<fake_id>"
         
         async_create = self.cs_client.container_services.create_or_update(
             resource_group.name,
             container_name,
             {
                 'location': location,
                 "orchestrator_profile": {
-                    "orchestrator_type": "DCOS"
+                    "orchestrator_type": "Swarm"
```

Comment:
Maybe change this to "Kubernetes"? I'm not sure how long "Swarm" will be supported for.

If it works for "Swarm", we can keep as is.